src/context/volume-context.js

```diff
@@ -0,0 +1,82 @@
+import {createContext, useContext, useReducer} from "react";
+
+const VolumeContext = createContext();
+
+// Custom component to provide the Volume context
+function VolumeProvider(props) {
+  const [state, dispatch] = useReducer(volumeReducer, {
+    toggles: {
+      measurement: 0,
+      season: 0, 
+      tide: 0,
+    },
+    path: "./assets/models/summer_low_salt.png",
```

Comment:
path should just be the path prefix, the full path should always be computed from the current state

Reply:
So initialize it as `./assets/models/`? 

I haven't figured out how to compute the path on initial render. Right now it's only called when the toggle buttons are changed.

Reply:
```suggestion
    path: `${pathPrefix}summer_low_salt.png`,
```

No worries! Seems like that will become easier once more components are functional